doc/plugins/org.polarsys.capella.core.ui.intro/src/org/polarsys/capella/core/ui/intro/views/WelcomeView.java

```diff
@@ -0,0 +1,314 @@
+
+    final Canvas viewCanvas = new Canvas(intermediaryComposite, SWT.FILL);
+    GridData viewCanvasLayoutData = new GridData();
+    viewCanvasLayoutData.horizontalAlignment = SWT.FILL;
+    viewCanvasLayoutData.heightHint = 180;
+    viewCanvasLayoutData.grabExcessHorizontalSpace = true;
+    viewCanvas.setLayoutData(viewCanvasLayoutData);
+
+    ImageDescriptor imgDesc = AbstractUIPlugin.imageDescriptorFromPlugin(EXTENSION_POINT_ID, "icons/ov_banner.jpg");
+    try {
+      banner_icon = imgDesc.createImage();
+    } catch (Exception e) {
+
+    }
+    final Image finalIcon = banner_icon;
+    ImageDescriptor imgDesc2 = AbstractUIPlugin.imageDescriptorFromPlugin(EXTENSION_POINT_ID,
+        "icons/firststeps-select.png");
+
+    try {
+      compass_icon = imgDesc2.createImage();
+    } catch (Exception e) {
+
+    }
+
+    final Image finalIcon2 = compass_icon;
+
+    viewCanvas.addListener(SWT.Paint, e -> {
+      GC gc = e.gc;
+
+      Rectangle rect = finalIcon.getBounds();
+      Rectangle client = viewCanvas.getClientArea();
+
+      gc.drawImage(finalIcon, rect.x, rect.y, rect.width, rect.height, client.x, client.y, client.width, rect.height);
+      gc.drawImage(finalIcon2, 10, 20);
+      gc.setBackground(e.display.getSystemColor(SWT.COLOR_WHITE));
+      gc.fillRectangle(rect.x, rect.height, client.width, client.height - rect.height);
+      // gc.setAlpha(0);
+      titleTextFont = new Font(e.display, "Verdana", 21, SWT.NORMAL);
+
+      gc.setFont(titleTextFont);
+      gc.drawText("First Steps", 75, 90, true);
+
+      introTextFont = new Font(e.display, "Verdana", 10, SWT.NORMAL);
+
+      gc.setFont(introTextFont);
+      gc.drawText("Getting started with Capella ", 75, 130, true);
+
+    });
+
+    viewCanvas.setBackgroundMode(SWT.INHERIT_FORCE);
+
+    Composite itemsComposite = new Composite(intermediaryComposite, SWT.FILL);
+    itemsComposite.setBackgroundMode(SWT.INHERIT_FORCE);
+    itemsComposite.setBackground(parent.getDisplay().getSystemColor(SWT.COLOR_WHITE));
+
+    GridData itemsCompositeLayoutData = new GridData();
+    itemsCompositeLayoutData.horizontalAlignment = SWT.FILL;
+    itemsCompositeLayoutData.verticalAlignment = SWT.FILL;
+    itemsComposite.setLayoutData(itemsCompositeLayoutData);
+
+    GridLayout itemsLayout = new GridLayout();
+    itemsComposite.setLayout(itemsLayout);
+    itemsLayout.numColumns = 2;
+
+    Composite leftPanel = new Composite(itemsComposite, SWT.FILL);
+    leftPanel.setBackgroundMode(SWT.INHERIT_FORCE);
+    leftPanel.setBackground(parent.getDisplay().getSystemColor(SWT.COLOR_WHITE));
+    GridLayout leftLayout = new GridLayout();
+    leftPanel.setLayout(leftLayout);
+    leftLayout.numColumns = 1;
+
+    Composite rightPanel = new Composite(itemsComposite, SWT.FILL);
+    rightPanel.setBackgroundMode(SWT.INHERIT_FORCE);
+    rightPanel.setBackground(parent.getDisplay().getSystemColor(SWT.COLOR_WHITE));
+    GridLayout rightLayout = new GridLayout();
```

Comment:
You can use a spacing and margins to add some surrounding paddding
    GridLayout leftLayout = GridLayoutFactory.fillDefaults().spacing(new Point(30, 30)).margins(new Point(30, 30)).create();